sdk/communication/azure-communication-callautomation/src/main/java/com/azure/communication/callautomation/models/CollectTonesResult.java

```diff
@@ -36,4 +37,16 @@ public CollectTonesResult setTones(List<DtmfTone> dtmfTones) {
         this.dtmfTones = dtmfTones;
         return this;
     }
+
+    /**
+     * Set the tones property: The tones property.
+     *
+     * @return the CollectTonesResult object itself.
+     */
+    public String convertToString() {
+        return dtmfTones.stream()
+                .map(x -> x.toChar())
```

Comment:
why is needed to convert to char? why not to to string directly and you remove one step?

Reply:
it is a naming that the converted type is an element in char format, we could convert it to string directly, but the naming would be toString or convertToString, which doesn't reflect the motivation of the change purpose.

Reply:
> which doesn't reflect the motivation of the change purpose.
I thought that the change purpose is to create a string out of the CollectTonesResult object and `toChar` was just an intermediate operation. If you get rid of `toChar` and call directly `toString` you do the same work with one step less.